phpstan.neon.dist

```diff
@@ -5,3 +5,5 @@ parameters:
 	paths:
 		- src
 		- tests
+	ignoreErrors:
```

Comment:
Add this to baseline instead :)

Reply:
What do you mean?

Reply:
I meant separate baseline file.. but nevermind